src/tools/kotlin-entity-generator.ts

```diff
@@ -323,8 +282,8 @@ var ${escaped}: ${type}
         }`;
   }
 
-  private mapFields(mapper: (arg: KotlinSchemaField) => string): string[] {
-    return this.entityDescriptor.fields.map(mapper);
+  private mapFields(mapper: (arg: ExtendedSchemaField) => string): string[] {
+    return this.fields.map(mapper);
```

Comment:
Now that this function is so simple (no property lookup), is it really needed?

Reply:
Indeed! Thanks! s/mapField/fields.map/ woohoo!